src/main/java/com/datapipe/jenkins/vault/VaultBuildWrapper.java

```diff
@@ -86,7 +85,7 @@ public VaultBuildWrapper(@CheckForNull List<VaultSecret> vaultSecrets) {
 
     @Override
     public void setUp(Context context, Run<?, ?> build, FilePath workspace,
-        Launcher launcher, TaskListener listener, EnvVars initialEnvironment) {
+                      Launcher launcher, TaskListener listener, EnvVars initialEnvironment) {
```

Comment:
style config is located here: https://github.com/jenkinsci/hashicorp-vault-plugin/tree/master/.idea/codeStyles

Reply:
uuh, next line of parameters on the same indentation level as first code line of the methods 
but 🤷‍♂ its your code... 😅